webtool/test_ytdlp_update.py

```
@@ -53,6 +53,18 @@ def isoliert(monkeypatch, tmp_path):
     return tmp_path
 
 
+def _warte_bis(bedingung, frist=5.0):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_warte_bis`

Add return type annotation: `bool`

(ANN202)